src/components/template-tab.js

```diff
@@ -1,137 +1,39 @@
 'use strict';
 
-const _ = require('lodash');
 const React = require('react');
+const { createContainer } = require('sovereign');
 
-const client = require('../lib/client');
 const Button = require('../primed/button');
-const chromeApi = require('chromeback')(chrome);
-const getErrorMessage = require('../lib/getErrorMessage');
+const templateStore = require('../stores/templateStore');
+const {
+  fetchStoredTemplate,
```

Comment:
i'm wondering if these actions shouldn't all be together, a store can listen to actions generated in multiple files.  Try breaking these actions up into more logical groups

This will also help to keep the definition on 1 line